rustls/src/server/server_conn.rs

```diff
@@ -799,6 +799,21 @@ mod connection {
                 sig_schemes,
             }))
         }
+
+        /// Yields an iterator over the buffered chunks of bytes
+        ///
+        /// Will yield an empty iterator after `accept()` has returned `Ok(Some(_))` or `Err(_)`.
+        pub fn buffered(&self) -> impl Iterator<Item = &[u8]> {
+            self.inner
+                .iter()
+                .map(|conn| {
+                    conn.core
+                        .common_state
+                        .sendable_tls
```

Comment:
Is this the right object? If I understood the purpose of this correctly, shouldn't this be `deframer_buffer`?

Reply:
🤦 You're right, of course. Still pretty straightforward...